### PR TITLE
Add security support for Dropwizard

### PR DESCRIPTION
**Pull request by kelnos**
**Contributing to Twilio**

> All third party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.

- [ ] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: modules/codegen/src/main/scala/com/twilio/guardrail/generators/Java/DropwizardServerGenerator.scala [hunk 1]
```diff
@@ -124,6 +131,112 @@ object DropwizardServerGenerator {
...
+                    val parameterType    = if (rawParameterType.isOptional) rawParameterType else optionalType(rawParameterType)
+                    val handlerParameter = new Parameter(util.EnumSet.of(FINAL), parameterType, new SimpleName(name.toCamelCase))
+                    val routeParameter = handlerParameter
+                      .clone()
+                      .addAnnotation(new SingleMemberAnnotationExpr(new Name(annotationName), new StringLiteralExpr(name)))
+                    SecurityParameters(List(routeParameter), List.empty, List(handlerParameter), List(new NameExpr(name.toCamelCase)))
+                  }
+
+                case scheme @ HttpSecurityScheme(authScheme, _) =>
+                  authScheme match {
```
Comment (blast-hardcheese) [inline modules/codegen/src/main/scala/com/twilio/guardrail/generators/Java/DropwizardServerGenerator.scala]: case sensitivity?

Comment (kelnos) [reply modules/codegen/src/main/scala/com/twilio/guardrail/generators/Java/DropwizardServerGenerator.scala]: The spec doesn't seem to suggest anything but lowercase is allowed.

**Comment by kelnos** [comment]
Note that it's not 100% critical that this one go into the next release; if it's getting to be too much to review all these, I'm fine with this one missing the cut-off.

**Comment by blast-hardcheese** [comment]
we do have a lot of changes slated for this release. I'm interested in #222 and #259, as @tomasherman has been working diligently on those for a while. If we can wrap up the rest of the stuff that's bugfix, that'd be appreciated before starting anew on new feature work.

**Comment by kelnos** [comment]
Realized I've been doing this the wrong way for DW.  Gonna close this until I have it worked out.

**Pull request closed without merging**